docs/source/metrics.rst

```diff
@@ -1,137 +1,78 @@
-The main difference between batch and epoch metrics is that **batch metrics** are computed at each batch and averaged at the end of an epoch whereas **epoch metrics** compute statistics for each batch and compute the metric at the end of the epoch.
-**Torch metrics** can be passed either as epoch metrics (only displayed at the end of the epoch) or using the specialized :class:`torch_metrics argument <poutyne.Model>` (displayed for each batch and at the end of the epoch).
-
-Epoch metrics offer a way to compute metrics that are not decomposable as an average.
-For instance, the predefined :class:`~poutyne.F1` epoch metric implements the F1 score.
-One who knows what an F1-score may know that an average of multiple F1 scores is not the equivalent of the overall F1 score.
-
-Here is an example using the three kinds of metrics:
-
-.. code-block:: python
-
-  from poutyne import Model, Accuracy, F1
-  import torchmetrics
+Poutyne offers two kinds of metrics: batch and epoch metrics.
+The main difference between batch and epoch metrics is that **batch metrics** are computed at each batch whereas **epoch metrics** compute statistics for each batch and compute the metric at the end of the epoch.
+Batch metrics are passed to :class:`~poutyne.Model` and :meth:`ModelBundle.from_network() <poutyne.ModelBundle.from_network()>` using the ``batch_metrics`` argument.
+Epoch metrics are passed to :class:`~poutyne.Model` and :meth:`ModelBundle.from_network() <poutyne.ModelBundle.from_network()>` using the ``epoch_metrics`` argument.
 
-  model = Model(
-      network,
-      'sgd',
-      'cross_entropy',
+In addition to the predefined metrics below, all PyTorch loss functions can be used by string name under their `functional <https://pytorch.org/docs/stable/nn.functional.html#loss-functions>`_ name.
+The key in :class:`callback logs<poutyne.Callback>` associated with each of them is the same as its name but without the ``_loss`` suffix. For example, the loss function :func:`~torch.nn.functional.mse_loss` can be passed as a metric with the name ``'mse_loss'`` or simply ``'mse'`` and the keys are going to be ``'mse'`` and ``'val_mse'`` for the training and validation MSE, respectively.
+Note that you can also pass the PyTorch loss functions as a loss function in :class:`~poutyne.Model` in the same way.
 
-      batch_metrics=[Accuracy()],
-      # Can also use a string in this case:
-      # batch_metrics=['accuracy'],
+.. warning:: When using the ``batch_metrics`` argument, the metrics are computed for each batch.
+    This can significantly slow down the compuations depending on the metrics used.
+    This mostly happens on non-decomposable metrics such as :class:`torchmetrics.AUROC <torchmetrics.AUROC>` where an ordering of the elements is necessary to compute the metric.
+    In such case, we advise to use them as epoch metrics instead.
```

Comment:
```suggestion
.. warning:: When using the ``batch_metrics`` argument, the metrics are computed for each batch.
    This can significantly slow down the computations depending on the metrics used.
    This mostly happens on non-decomposable metrics such as :class:`torchmetrics.AUROC <torchmetrics.AUROC>` where an ordering of the elements is necessary to compute the metric.
    In such a case, we advise using them as epoch metrics instead.
```